run_dir/design/projects.html

```diff
@@ -273,6 +273,23 @@ <h4 class="search-action" data-action="filterHeader" id="undefined">Undefined <i
               </div>
             </div>
           </div>
+          <div class="form-group row row-eq-height">
+            <div class="col-md-12 panel-group">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h4 class="panel-title">
+                    Reorder Table Fields
+                  </h4>
+                </div>
+                <div class="panel-body">
+                  <div class="sortableList" id="tHeaderList">
+                      <ul class="source list-inline" id="tHeaderListul">
```

Comment:
Is source a class that you chose? It sounds a bit generic can you have something slightly more specific?

Reply:
Done! 👍